tile-rendering/src/main/java/com/oculusinfo/tile/rendering/transformations/tile/BucketTileTransformer.java

```diff
@@ -80,15 +85,19 @@ public BucketTileTransformer(JSONObject arguments){
 			minimumValue = _minVals.get(_startBucket);
 			maximumValue = _maxVals.get(_startBucket);
 		} else {
-			for (int i = _startBucket; i < _endBucket; i++) {
-				Double val = _minVals.get(i);
-				if (val < minimumValue) {
-					minimumValue = val;
-				}
-				val = _maxVals.get(i);
-				if (val > maximumValue) {
-					maximumValue = val;
+			if (_endBucket <= _minVals.size()) {
```

Comment:
This causes the extema to be 0 when too many buckets are requested... We might want to instead return what buckets do exist, instead of returning nothing (i.e. for i = starbucket to min(endBucket, minVals.size))